ui/api-client/dist/esm/runtime.d.ts

```diff
@@ -157,8 +157,8 @@ export interface ApiResponse<T> {
     value(): Promise<T>;
 }
 export interface VoidResponse {
-    auth: unknown;
-    data: unknown;
+    auth: null;
+    data: null;
```

Comment:
Something is off here. I updated this in the generator template but maybe didn't push the commit. Let me check on that but the type should be `unknown`.